src/frontend/src/btc/components/BtcTransactions.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import Header from '$lib/components/ui/Header.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+</script>
+
+<Header>
+	<h2 class="text-base">{$i18n.transactions.text.title}</h2>
```

Comment:
That's actually an error in `IcTransactions` page which I guess you used as inspiration to create this page. It should be just the text - no `h2`.

Can you modify it and also provide a fix for the other page if you do not mind?

Reply:
Done!